READMEquay.md

```diff
@@ -0,0 +1,89 @@
+# Building with quay.io
+
+## quay.io repositories
+
+quay.io is a convenient place to build and store your development 
+docker images.  Set up an account for yourself and then set up
+an automatic build of your ovn-kubernetes fork. The build is
+triggered  whenever you push changes to your fork. The build is
+on the master branch.  The result is
+```
+quay.io/repository/<your-account>/ovn-kube:latest
+```
+You can use the image in your daemonsets for development.
+
+Dockerfile.centos builds the binaries and then includes them in the 
+image.
+
+
+## setup docker pull access on each host in the cluster
```

Comment:
I'm not sure this is necessary if you use a full URL (e.g. quay.io/foo/bar)

Reply:
@squeed I deleted the section